go.mod

```diff
@@ -5,9 +5,11 @@ go 1.22.0
 toolchain go1.23.0
 
 require (
+	github.com/Javex/caddy-fail2ban v0.0.0-20240525135356-cbf9f5a003ce
 	github.com/caddyserver/caddy/v2 v2.8.4
 	github.com/mastercactapus/proxyprotocol v0.0.4
 	github.com/miekg/dns v1.1.62
+	github.com/quic-go/quic-go v0.44.0
```

Comment:
Seems like I failed to update go.mod within one of the latest QUIC PRs. That's why it's visible as an addition here.